src/System.Numerics/ref/Compat/System.Numerics.cs

```diff
@@ -27,7 +27,7 @@ public partial struct BigInteger : System.IComparable, System.IComparable<System
         public static int Compare(System.Numerics.BigInteger left, System.Numerics.BigInteger right) { return default(int); }
         public int CompareTo(long other) { return default(int); }
         public int CompareTo(System.Numerics.BigInteger other) { return default(int); }
-        public int CompareTo(object obj) { return default(int); }
+        int System.IComparable.CompareTo(object obj) { return default(int); }
```

Comment:
Yup, not sure why I haven't closed this already. I'll close it and send out a new PR that only contains new members that were exposed.